daily-challenges/2024-09/19/hunghoang3110.spec.ts

```
@@ -0,0 +1,42 @@
+let productList = {}
+function addProduct (name, price) {
+    productList[name] = price;
+}
+
+function removeProduct (name) {
+    delete productList[name]
+}
+
+function caculateTotal () {
+    let total = 0;
+    for (let [product, value] of Object.entries(productList)) {
```

Biome (1.9.4)

[error] This let declares some variables that are only assigned once.

'product' is never reassigned.

'value' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)